### PR TITLE
fixed comma in greeting, fixed build error, made logout button chang…

### PR DESCRIPTION
**Pull request by dwee9**
…e to log in when user is not signed in

### CODE REVIEW

File: Droid/Fragments/HomeFragment.cs [hunk 1]
```diff
@@ -126,7 +126,7 @@ public override void OnResume( )
 
                 if( !string.IsNullOrEmpty( firstName ) )
                 {
-                    welcomeText += string.Format( ", {0}", firstName );
+                    welcomeText += string.Format( "{0}", firstName );
```
Comment (jhawkzz) [inline Droid/Fragments/HomeFragment.cs]: This doesn't need to be a `string.Format` anymore

File: App.Shared/Config.cs [hunk 2]
```diff
@@ -34,7 +34,7 @@ public static class GeneralConfig
         /// </summary>
 #if DEBUG
 #if __ANDROID__ //Android won't allow https with self signed certs, so don't use https for dev.
-        public const string RockBaseUrl = "http://ma3dev.ccv.church/";
+        public const string RockBaseUrl = "https://ma3dev.ccv.church/";
```
Comment (jhawkzz) [inline App.Shared/Config.cs]: This needs to remain `http` for development

Comment (jhawkzz) [reply App.Shared/Config.cs]: Or, actually, if we've verified it works fine with `https`, lets remove the #ifdef for Android and comment.

Comment (dwee9) [reply App.Shared/Config.cs]: I tried it on a device and the simulator and it worked for me

File: Droid/Droid.csproj [hunk 3]
```diff
@@ -33,7 +33,7 @@
     <ConsolePause>false</ConsolePause>
     <AndroidEnableMultiDex>true</AndroidEnableMultiDex>
     <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
-    <AndroidSupportedAbis>armeabi-v7a;x86;arm64-v8a;x86_64</AndroidSupportedAbis>
```
Comment (jhawkzz) [inline Droid/Droid.csproj]: Intentionally removing this compatibility?

Comment (jhawkzz) [reply Droid/Droid.csproj]: Intentionally removing this compatibility?

Comment (jhawkzz) [reply Droid/Droid.csproj]: Sorry for the double post!

Comment (dwee9) [reply Droid/Droid.csproj]: Ah, this was the reason it wasn't deploying for me, i did not mean to include it

**Pull request merged**